README.md

```diff
@@ -1,8 +1,64 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# David Gordon . Jarvis Consulting
 
+Hi, my name is David Gordon. I am a professional Software Engineer based in Toronto, working on various projects at Jarvis. I received a diploma in computer programming from Conestoga College, and solved many problems along the way. I'm experienced in various technologies, languages and frameworks, creating efficient applications and elegant interfaces.
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git, C#
+
+**Competent:** C/C++, Rust, Typescript, Python, React
+
+**Familiar:** Azure, Angular, Assembly, Lua, Non-Relational DBs
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_DavidGordon](https://github.com/jarviscanada/jarvis_data_eng_DavidGordon)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_DavidGordon/tree/master/linux_sql)]: The Linux Cluster Monitoring Agent is a simple yet powerful solution to gathering a host system's hardware information and usage. This product is intended for IT teams who need a way to log and review their systems usage to manage server costs and performance.
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_DavidGordon/tree/master/core_java)]:
```

Comment:
Remove the placeholders when you get a chance, low priority but needs to be done before we get your profile out to clients for obvious reasons. Just be very careful when removing sections that you remove the whole object.